setup.py

```diff
@@ -38,7 +38,7 @@
 geospatial_requires = ['geoalchemy2', 'geopandas', 'shapely']
 
 dask_requires = [
-    'dask[dataframe, array]>=2021.1.1,<2021.5.1',
+    'dask[dataframe, array]>=2021.1.1,!=2021.5.*,!=2021.6.0',
```

Comment:
ok so we work with 2021.6.1 (and 2021.5.1), but not he .0, ok

Reply:
We work with anything `>` 2021.1.1, except for `2021.5.0, 2021.5.1, 2021.6.0`.